tests/console/system_prepare.pm

```diff
@@ -91,6 +91,12 @@ sub run {
         }
     }
 
+    # enable multiversion for kernel-default based on bsc#1097111, for migration continuous cases only
+    if (get_var('FLAVOR', '') =~ /Continuous-Migration/) {
+        script_run('sed -i s/#multiversion/multiversion/g /etc/zypp/zypp.conf');
```

Comment:
need a record_soft_failure for each change

Reply:
Update the code by adding record_soft_failure for each change, thanks. And I have re-run the VR.